docs/dev-machine-setup.md

```diff
@@ -15,7 +15,40 @@ outlined in the [Getting Started](getting-started.md) instructions).
 
 ### Local Virtual Machine
 
-`[TODO]`
+#### macOS
```

Comment:
I don't use a mac and I am not sure if this is the only way of enabling development on a mac. That's why I suggest updating the header or add a new heading with `Using VMWare Fusion` or something similar. With the current heading it might seem that the only supported way is VMware Fusion.

Reply:
From memory, the instructions seem correct to enable nested virt in VMWare. The steps for Parallels (another VM for macOS supporting nested virt) would be different. So it would be good to qualify that these instructions are for VMware Fusion.

Reply:
Changed the current heading to indicate these are just instructions for macOS + VMware Fusion. Figured the person who adds the second set of macOS instructions can rearrange the heading hierarchy to something like
```
### macOS

#### Using VMware Fusion

#### <second method>
```